web/impact/impact/urls.py

```diff
@@ -63,6 +64,8 @@
         name=JWTCookieNameView.view_name),
     url(r'^api/algolia/api_key/$', AlgoliaApiKeyView.as_view(),
         name=AlgoliaApiKeyView.view_name),
+    url(r'^api/calendar/reminder/$', CalendarReminderView.as_view(),
+        name='calendar_reminder_view'),
```

Comment:
this should be a field `view_name` on the CalendarReminderView class, and that field should be used by the test cases.

Reply:
fixed